modules/rostests/apitests/shell32/SHAppBarMessage.cpp

```diff
@@ -10,6 +10,9 @@
 #include <shlwapi.h>
 #include <stdio.h>
 
+#define NDEBUG
+#include <debug.h>
+
```

Comment:
```suggestion
```
These shouldn't be needed because we should have the `trace` macro available for tests.